engines/mysql.py

```diff
@@ -64,10 +68,13 @@ def insert_data_from_file(self, filename):
 IGNORE """ + str(self.table.header_rows) + """ LINES
 (""" + columns + ")"
             try:
+                self.cursor.execute(mysql_set_autocommit_off)
                 self.cursor.execute(statement)
+
+                self.cursor.execute(mysql_set_autocommit_on)
             except Exception as e:
-                print "Failed bulk insert (%s), inserting manually" % e
                 self.disconnect()  # If the execute fails the database connection can get hung up
+                self.cursor.execute(mysql_set_autocommit_on)
```

Comment:
in case of any exception, remember to set the configurations back ``self.cursor.execute(mysql_set_autocommit_on)``